tests/Integration/DataManagementSystem/Test_UserMetadata.py

```diff
@@ -0,0 +1,138 @@
+"""
+Test of multi-VO user metadata handling. Assumes a running Dirac instance with the (master?) FileCatalog
+"""
```

Comment:
All integration tests will be run automatically within GitHub Actions when they are added to either `DIRAC/tests/Integration/all_integration_client_tests.sh` or  `DIRAC/tests/Integration/all_integration_server_tests.sh`. So this one would need to be added in the `all_integration_client_tests.sh` file.

Reply:
While I was looking into adding a new Catalog with a correct configuration so multi VO metadata classes are loaded I looked into the tests which are currently ran on push. I can see dirac installation tests being run on my last commit but the integration tests section is empty. Am I looking into the wrong place ? (the green tick next to the commit number). Are they not to be run at each push ?